content/ai/concepts/generative-ai/generative-ai.md

```diff
@@ -0,0 +1,117 @@
+
+### The Question of Intelligence?
+
+Artificial Intelligence (AI) is the matching or exceeding the intelligence of a human being. Things that are often associated with intelligence such as doing math, playing chess, or remembering vast amounts of data already can be done more quickly and better by computer.
+
+With LLMs and a technology called diffusion AI, systems can now match or exceed other skills we often associate with intelligence such as:
+
+- Understanding and creating new text content.
+- Writing never before created stories and poems.
+- Analyzing content for tone and emotions.
+- Creating unique images and video.
+
+### Is It Art?
+
+In 2022 Jason Allen’s AI generated work, _Théâtre D’opéra Spatial_, won first place in the digital category at the Colorado State Fair. The artist did indicate that he used the GenAI platform Midjourney. It was later discovered that the judges weren't familiar with the tool at the time of judging.
+
+The question "Does GenAI create art?" is now being asked. When photography was introduced, only painting had been considered art. Photography wasn't considered art and the same discussion took place. Now, few people would argue that photography cannot be art.
+
+Another argument is that since there is no advanced skill required from the operator, it is not art. Some consider art as the application of skills developed over many years. However, some modern art has been about concept and technique - not necessarily skill.
+
+An additional argument against it being art is that that the human element is not present in GenAI created works. The counter argument is that the creation is still driven by a person who has emotion, creativity, and intent.
+
+## Applications
+
+Generative AI is a combination of several technologies and has many applications. Effective LLMs are used to understand a natural language prompt and technologies such as diffusion models are used in the case of image generation.
+
+Some content types GenAI is being used for include:
+
+- [Text](https://www.codecademy.com/resources/docs/ai/generative-ai/text)
+- [Images](https://www.codecademy.com/resources/docs/ai/generative-ai/images)
+- [Music](https://www.codecademy.com/resources/docs/ai/generative-ai/music)
+- [Code](https://www.codecademy.com/resources/docs/ai/generative-ai/code)
+
+### Other Areas
+
+While GenAI is generally discussed in the context of text, images, music, and coding, there are many other active areas. Some of these include:
+
+- High fidelity text to speech
+- Scientific Discovery - generating hypothesis, accelerating discovery
+- Chemistry - molecule and drug discovery
+- Medical imaging
+- 3D Models - text to space creation
+- Video - movies, animations, transition between real and generated
+- Education - personalizing, customizing lessons, tutoring
+- Many others!
+
+## Impact and Possibilities
+
+### Legality
```

Comment:
This section addresses the problematic use of material (without permission or credit) for training, but it doesn't really touch on one of the biggest issues related to AI, which is the creation of content that is inaccurate (due to biases in training data or other factors), or intentionally  deceptive (deepfakes, fake news content). Please expand this section or add another to include these areas of concern.

Reply:
Added an Accuracy section "The correctness of the input data used during model training controls the accuracy of the generated content. There is no way to guarantee quality, accuracy, lack of bias, or timeliness. With the increased consumption of generated content, cases of misinformation (unintended and intended) are being considered."